tests.py

```diff
@@ -7,6 +8,207 @@
 import utils
 import yaml
 
+import pytest
+
+import numpy as np
+
+from tensorflow.keras.datasets import cifar10,mnist
+from datasets import dataset_cifar10 as data_cf
+from datasets import dataset_mnist as data_mn
+from partner import Partner
+from dataset import Dataset
+from scenario import Scenario
+from multi_partner_learning import MultiPartnerLearning
+from pathlib import Path
+
+
+@pytest.fixture(scope="module", params=["a","b"])
```

Comment:
Need space in the params!

Reply:
And have you seen the `paramatrize`: 
```python
@pytest.mark.parametrize("a,b,expected", testdata, ids=["forward", "backward"])
def test_timedistance_v1(a, b, expected):
    diff = a - b
    assert diff == expected
```
It might be interesting if you want to mimic some configs: https://docs.pytest.org/en/latest/example/parametrize.html#paramexamples